Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.22 AS builder
+FROM golang:1.23 AS builder
```

Comment:
This change is tricky.
I think we need to use  renovate in this kind of changes